tests/fixtures/test_many_suppliers_many_lots_result.csv

```diff
@@ -0,0 +1,5 @@
+supplier_id,supplier_dm_name,application_status,declaration_status,draft_saas,completed_saas,draft_test_lot,completed_test_lot,draft_test_lot2,completed_test_lot2
+123,Bens cool supplier1,,completed,0,6,0,0,0,0
```

Comment:
These fixtures should have the `application_status` values filled in with `application`/`no application`.